DirectEmployers/MyJobs
# [PD 2891] cbo rules crud ui

## What's this PR do?
This PR creates the rules view and crud UI for the rules inside of CBO

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the CBO app. The main page has a page for rules along with seeing the rules for specific contact.

## How should this be manually tested?
Pull down the repository. Once inside of the CBO application, you will have the options for the rules view on the right hand side of the page along with having the options to see and use the CRUD functionality for the rules on a specific contact.

## Any background context you want to provide?
No need to merge right now. Might need some backend code to be merged inside of this so I am creating a pull request right now just in case if it needs to be pulled down for the backend support for the rules. PR is not completely finished.
